src/action-ros-ci.ts

```diff
@@ -304,16 +304,24 @@ async function run() {
 					colconCommandPrefix += `source ${ros2SetupPath} && `;
 				}
 			}
+		} else if (isWindows) {
+                        // Windows only supports ROS2
+			if (targetRos2Distro) {
+                                const ros2SetupPath = `C:\dev\${targetRos2Distro}\local_setup.bat`;
+				if (fs.existsSync(ros2SetupPath)) {
+					colconCommandPrefix += `call ${ros2SetupPath} && `;
+				}
+			}
 		}
-
+		
```

Comment:
Remove whitespace at EOL. 

I think there are mixed tabs and spaces on a few above lines as well which is what causes the misalignment on github rendering - could you convert to all spaces?

Reply:
oops yeah sorry

Reply:
I think it's actually tabs and not spaces